### PR TITLE
Use roles, dude. 

### PR DESCRIPTION
**Pull request by brucellino**
See #3

### CODE REVIEW

File: plantisc2/tasks/db.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
```
Comment (mtorrisi) [inline plantisc2/tasks/db.yml:1]: For this role @brucellino suggested me to check something in Galaxy, I'm having a look at [this](https://galaxy.ansible.com/geerlingguy/mysql/)

File: plantisc2/tasks/nginx-config.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    state: directory
+    owner: www-data
+    group: www-data
+    mode: 0700
+
+- name: Configure nginx
+  template:
+    src: nginx.conf.j2
+    dest: /etc/nginx/sites-available/default
+  notify:
```
Comment (mtorrisi) [inline plantisc2/tasks/nginx-config.yml]: Can we reuse the handlers roles to restart services?

File: plantisc2/tasks/php-config.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+- name: Ensure php7.0-fpm cgi.fix_pathinfo=0
+  lineinfile:
+    dest: /etc/php/7.0/fpm/php.ini
+    regexp: '^(.*)cgi.fix_pathinfo='
+    line: cgi.fix_pathinfo=0
+  notify:
```
Comment (mtorrisi) [inline plantisc2/tasks/php-config.yml:7]: Can we reuse the handlers roles to restart services?

File: plantisc2/tasks/prerequisites.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+- package:
+    name: "{{ item }}"
+    update_cache: yes
+    state: "latest"
+  with_items: "{{ prerequisite_packages[ansible_os_family] }}"
+  notify:
```
Comment (mtorrisi) [inline plantisc2/tasks/prerequisites.yml]: Can we reuse the handlers roles to restart services?

File: plantisc2/tasks/nginx-config.yml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    - restart php7.0-fpm
+    - restart nginx
+
```
Comment (mtorrisi) [inline plantisc2/tasks/nginx-config.yml:17]: Can we reuse the handlers roles to restart services?

File: plantisc2/tasks/prerequisites.yml [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+    - restart php7.0-fpm
+    - restart nginx
```
Comment (mtorrisi) [inline plantisc2/tasks/prerequisites.yml:9]: Can we reuse the handlers roles to restart services?

File: plantisc2/tasks/db.yml (hunk 1 above)
Comment (brucellino) [reply plantisc2/tasks/db.yml:1]: yes, that would be a good suggestion. But let's keep the code in the project and not use submodules for now.

File: plantisc2/tasks/php-config.yml (hunk 3 above)
Comment (brucellino) [reply plantisc2/tasks/php-config.yml:7]: yes, as long as it's within the same role, you can call the handlers from any task.

File: plantisc2/tasks/nginx-config.yml (hunk 5 above)
Comment (brucellino) [reply plantisc2/tasks/nginx-config.yml:17]: yes, as long as the task is in the same role, you can trigger the handler.

File: plantisc2/tasks/prerequisites.yml (hunk 6 above)
Comment (brucellino) [reply plantisc2/tasks/prerequisites.yml:9]: yes that's the whole point of handlers.

**Comment by brucellino** [comment]
Can merge.

**Pull request merged**